browserstack_examples_specflowplus/BrowserStack/src/stepdefs/e2e/e2eSteps.cs

```diff
@@ -0,0 +1,124 @@
+using TechTalk.SpecFlow;
+using OpenQA.Selenium;
+using System;
+using BrowserStack.WebDriver.Core;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium.Chrome;
+//using SeleniumExtras.WaitHelpers;
+using NUnit.Framework;
+using SpecflowBrowserStack.src.stepdefs;
+
+namespace SpecflowBrowserStack.Steps
+{
+	[Binding]
+	public class e2eSteps

```

Comment:
can you follow the all the C# conventions for ClassName, instance / local variable names etc. throughout the project

https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions